src/posit/connect/resources.py

```diff
@@ -223,31 +171,25 @@ def reload(self) -> "ActiveSequence":
         return self
 
 
-class ActiveFinderMethods(ActiveSequence[T_co], ABC, Generic[T_co]):
-    """
-    Finder methods.
+class ActiveFinderMethods(ActiveSequence[T], ABC, Generic[T]):
+    def __init__(self, ctx: Context, parent: Optional[Active] = None, uid: str = "guid"):
+        """Finder methods.
 
-    Provides various finder methods for locating records in any endpoint supporting HTTP GET requests.
+        Provides various finder methods for locating records in any endpoint supporting HTTP GET requests.
```

Comment:
When we get to quartodoc updates, we will most likely need to update where these docs are located. But it should just be shuffling them around and chopping type definitions.